igor-web/src/main/groovy/com/netflix/spinnaker/igor/gitlabci/client/GitlabCiClient.java

```diff
@@ -30,12 +32,32 @@
   List<Project> getProjects(
       @Query("membership") boolean limitByMembership,
       @Query("owned") boolean limitByOwnership,
-      @Query("page") int page);
+      @Query("page") int page,
+      @Query("per_page") int pageLimit);
+
+  @GET("/api/v4/projects/{projectId}")
+  Project getProject(@Path("projectId") String projectId);
 
   @GET("/api/v4/projects/{projectId}/pipelines")
-  List<PipelineSummary> getPipelineSummaries(
-      @Path("projectId") int projectId, @Query("per_page") int pageLimit);
+  List<Pipeline> getPipelineSummaries(
+      @Path("projectId") String projectId, @Query("per_page") int pageLimit);
 
   @GET("/api/v4/projects/{projectId}/pipelines/{pipelineId}")
-  Pipeline getPipeline(@Path("projectId") int projectId, @Path("pipelineId") long pipelineId);
+  Pipeline getPipeline(@Path("projectId") String projectId, @Path("pipelineId") long pipelineId);
+
+  @GET("/api/v4/projects/{projectId}/jobs/{jobId}/artifacts/{artifactPath}")
+  Response getArtifactFile(
```

Comment:
Is this used anywhere?

Reply:
It was but I hit the blocker in my main post, i'll try reintroduce in another PR that has artifact support

Reply:
I'd say remove it from this PR if nothing is using it.

Reply:
Sorry...you've already removed it.